src/hotspot/share/runtime/registerMap.hpp

```diff
@@ -70,6 +70,9 @@ class RegisterMap : public StackObj {
     location_valid_type_size = sizeof(LocationValidType)*8,
     location_valid_size = (reg_count+location_valid_type_size-1)/location_valid_type_size
   };
+  enum class UpdateMap { skip, yes };
+  enum class ProcessFrames { skip, yes };
+  enum class WalkContinuation   { skip, yes };
```

Comment:
Instead of `yes` I would recommend using like `include` (or `add') as `yes` seems relatively unspecific compared to `skip`.

Reply:
`include` is probably better. I was trying to think of a good word to use, first I thought of `do` and `skip` because the enums are actions/verbs. But the reverse order (as in `UpdateMap :: do`) sounds a bit weird. Thought of it more as a question instead, `Action? :: yes/skip`. But `include` fits better with respects to `skip`, `yes`/`no` is an alternative. But `skip`/`include` is nicer. Also an excuse to fix that double space on line 75.